projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html

```diff
@@ -29,7 +29,7 @@
           [fitToContainer]="true"
           [content]="getCardContent(address) | async"
           (editCard)="editAddressButtonHandle(address)"
-          (setDefaultCard)="setAddressAsDefault(address.id)"
+          (setDefaultCard)="setAddressAsDefault(address)"
```

Comment:
Please mention about this change in migration doc.

Reply:
Added